TelePharo-Core.package/TlpRemoteUIManager.class/instance/cleanupEnvironmentFromRemoteTool.st

```diff
@@ -0,0 +1,8 @@
+tools registration
+cleanupEnvironmentFromRemoteTool
+		"This method resets all announcers Seamless. When the network is broken, the server keeps the old announcers, so we need reset them"
+	SystemAnnouncer uniqueInstance subscriptions subscriptions
+   select: [ :each | each subscriber isSeamlessProxy ]
+   thenDo: [ :each | SystemAnnouncer uniqueInstance unsubscribe: each subscriber].
```

Comment:
We still need to reset the Calypso query cache (the code was in previous comments)